pwa/src/templates/templateParts/resultsTemplates/layers/LayersResultTemplate.module.css

```diff
@@ -0,0 +1,41 @@
+.Container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.Container > * {
+  margin-block-end: var(--nlportal-space-block-lg);
+}
+
+.Container > *:not(:last-child) {
+  margin-inline-end: var(--nlportal-space-block-xl);
+}
+
+.Components {
+  display: inline-block;
+  background-color: rgb(245, 245, 245);
+  box-shadow: rgb(33 33 33 / 25%) 0px -3px 0px inset;
+  padding: 14px;
+}
+
+.Components > a {
+  text-decoration: none;
+}
+
+.beta {
+  border-right-width: 8px;
+  border-right-style: solid;
+  border-right-color: var(--conduction-color-beta);
+}
+
+.gepland {
+  border-right-width: 8px;
+  border-right-style: solid;
+  border-right-color: var(--conduction-color-gepland);
+}
+
+.gewenst {
+  border-right-width: 8px;
+  border-right-style: solid;
+  border-right-color: var(--conduction-color-gewenst);
+}
```

Comment:
I'd opt to combine these `border-right-width` and `border-right-style` rules in one block. Right now, if we would want to change the style or width, we have to update all these rules.

(I tested just adding it to `.component`, however, that created a black border on the unselected)

A possible solution would be;

```
.beta,
.gewenst,
.gepland {
  border-right-width: 8px;
  border-right-style: solid;
}
```